### ISSUE TITLE
Question about the corresponding code of loss function

### ISSUE THREAD

**Issue by godspeed1989** [ZiweiWangTHU/BiDet#3]
Hi, Wang. Thanks for your great work.
Can you specify the corresponding source code for your loss function in paper:
![image](https://user-images.githubusercontent.com/2350713/82006645-dc195880-969a-11ea-9fac-adee2db77392.png)

Thank you.

**Comment by Wuziyi616** [comment]
Hi! Thank you for your interest on our work! The formulation you posted above is derived from the information bottleneck, but in our implementation, we actually use MC sampling to approximate the distribution of the output.
That is to say, the network output \mu and \sigma of the distributions, and I calculate loss using l and c sampled from the distribution. So in the loss function I just use normal smooth-l1 loss and cross-entropy loss.
